pkg/plugin/conntrack/_cprog/conntrack.h

```diff
@@ -13,7 +13,7 @@
 	({ __u64 __x = bpf_ktime_get_boot_ns() / NSEC_PER_SEC; __x; })
 # define bpf_mono_now()		bpf_ktime_get_sec()
 
-#define UINT64_MAX 18446744073709551615ULL
+#define UINT32_MAX 4294967295U
```

Comment:
see if this works: https://stackoverflow.com/questions/56975861/error-compiling-ebpf-c-code-out-of-kernel-tree

Reply:
ok so this is the error if i try to include the header file with `UNIT32_MAX` 
![image](https://github.com/user-attachments/assets/f66d15f6-31bc-439b-8f45-d045681b9a25)